app/utils/helpers.py

```diff
@@ -48,3 +51,20 @@ def text_to_markdown(text):
                                 tags=ALLOWED_TAGS,
                                 attributes=ALLOWED_ATTRIBUTES)
     return sanitized_html
+
+
+def dp_in_readme(readme, dp):
+    """ This method takes a readme and data package as arguments. If there is dp
```

Comment:
think it actually takes metadata (datapackage.json) of data package as an argument and not data package itself, correct?